src/invoke_training/_shared/stable_diffusion/validation.py

```diff
@@ -20,6 +20,41 @@
 from invoke_training.pipelines.stable_diffusion_xl.lora.config import SdxlLoraConfig
 
 
+def split_prompt(prompt: str) -> tuple[str, str]:
```

Comment:
I *think* the `split_prompt()` logic can be replaced by a relatively simple regex pattern. Especially if we only allow `[...]` at the end of the input - which seems reasonable to me.

I'll loop back on this, unless you want to give it a shot.